packages/fxa-payments-server/package-lock.json

```diff
@@ -2905,6 +2905,12 @@
         "@types/node": "*"
       }
     },
+    "@types/classnames": {
+      "version": "2.2.8",
+      "resolved": "https://registry.npmjs.org/@types/classnames/-/classnames-2.2.8.tgz",
+      "integrity": "sha512-3UrLzPnz8u+MMXuJTF++389IfLSQUbl5F3ry9WCxva0BKG5H/oo5NuPRXk+HrpPU1+5pVHSWhnVWRzIaFQ7QuQ==",
+      "dev": true
+    },
```

Comment:
I regenerated this to fix https://github.com/mozilla/fxa/issues/1561, not sure if I did it right...

Reply:
It looks like that issue was fixed by https://github.com/mozilla/fxa/pull/1578, do you still need this?